src/protocols/permissions.ts

```diff
@@ -355,34 +365,38 @@ export class PermissionsProtocol {
     return scope.interface === 'Records';
   }
 
-
   /**
    * Validates scope.
    */
-  private static validateScope(scope: PermissionScope): void {
+  private static validateScope(scope: PermissionScope, grantRecord: RecordsWriteMessage): void {
     if (!this.isRecordPermissionScope(scope)) {
       return;
     }
+
     // else we are dealing with a RecordsPermissionScope
 
-    // `schema` scopes may not have protocol-related fields
-    if (scope.schema !== undefined) {
-      if (scope.protocol !== undefined || scope.contextId !== undefined || scope.protocolPath) {
-        throw new DwnError(
-          DwnErrorCode.PermissionsProtocolValidateScopeSchemaProhibitedProperties,
```

Comment:
Remove this error code from `DwnErrorCode`

Reply:
Good catch, done.